docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -453,3 +453,10 @@ After the shebang, the UTF-8 coding, the copyright line, the license, and the se
    from module_utils.basic import AnsibleModule
 
 The use of "wildcard" imports such as ``from module_utils.basic import *`` is no longer allowed.
+
+.. _testing_module_documenation:
```

Comment:
```suggestion
.. _testing_module_documentation:
```

Reply:
Corrected.

Reply:
sorry about the duplicate label error now.  Suggest something like _dev_testing_module_documentation  to make it unique.

Reply:
Done.